### PR TITLE
Update Changelog and README

### PR DESCRIPTION
**Pull request by OumaimaHamza**
Follow up to #27

### CODE REVIEW

File: README.md
Comment (pruzanov) [inline README.md:241]: Check for these leading extra spaces, they will prevent proper rendering of an .md file (you should check that README.md looks Ok on your branch before merging)

File: README.md
Comment (pruzanov) [inline README.md:243]: leading space should be removed

**Review by pruzanov — CHANGES_REQUESTED** [review]

**Review by pruzanov — APPROVED** [review]
Although there are some leading spaces, I see that github still can render README.md properly. I will leave it up to you - fix or keep it as is

**Comment by OumaimaHamza** [comment]
Since this was generated with the `generate_markdown_readme.py `script, I'm gonna keep as is. Thank you!

**Pull request merged**
